### PR TITLE
Assignment5 Submission (Minseok Lee)

### PR DESCRIPTION
**Pull request by mslee300**
Submitting Assignment 5

### CODE REVIEW

File: Minseok_Lee [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (aakankshaduggal) [inline Minseok_Lee:1]: file not reqd

**Comment by aakankshaduggal** [comment]
@mslee300 - Can you please add a jupyter notebook that has all the results and visualizations in one place?

**Comment by mslee300** [comment]
Add a jupyter notebook that has all the results and visualizations in one place as requested by Aakanksha

**Comment by mslee300** [comment]
I've added the jupyter notebook that has all the results and visualizations
in one place to the original PR, thank you!

**Comment by aakankshaduggal** [comment]
@mslee300 - thanks for updating!
Good work, I miss some documentation(-10) and explanation for your hypothesis(-10).
total - 80/100

**Pull request closed without merging**